src/sparseml/exporters/transforms/utils/helpers.py

```diff
@@ -125,22 +126,17 @@ def assert_node_type(node: NodeProto, op: Union[List[str], Set[str], str]) -> bo
 def quantize_array(
     array: numpy.ndarray, scale: float, zero_point: int, dtype: Any = numpy.uint8
 ) -> numpy.ndarray:
+    try:
+        from sparseml.exporters.transforms.utils.helpers_torch import (
+            quantize_array_torch,
+        )
 
-    if dtype == numpy.uint8:
-        tensor_dtype = torch.quint8
-    elif dtype == numpy.int8:
-        tensor_dtype = torch.qint8
-    elif dtype == numpy.int32:
-        tensor_dtype = torch.qint32
-
-    tensor = torch.Tensor(array.copy()).to(torch.float32)
-    if isinstance(scale, numpy.ndarray):
-        scale = scale.item()
-    if isinstance(zero_point, numpy.ndarray):
-        zero_point = zero_point.item()
-
-    quant_tensor = torch.quantize_per_tensor(tensor, scale, zero_point, tensor_dtype)
-    return quant_tensor.int_repr().numpy()
+        return quantize_array_torch(array, scale, zero_point, dtype)
```

Comment:
I think we should keep the logic for torch quantization in this file. It's not used anywhere else and it helps to keep number of files & levels of indirection down if we have it here.